neo/SDK/SC/Nep5API.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Neo.IO;
+using Neo.Network.P2P.Payloads;
+using Neo.Network.RPC;
+using Neo.SDK.TX;
+using Neo.VM;
+using Neo.Wallets;
+using System.Linq;
+using System.Numerics;
+
+namespace Neo.SDK.SC
+{
+    public class Nep5API : ContractClient
+    {
+        public Nep5API(RpcClient rpcClient) : base(rpcClient) { }
+
+        public BigInteger BalanceOf(UInt160 scriptHash, UInt160 account)
+        {
+            BigInteger balance = TestInvoke(scriptHash, "balanceOf", account).Stack.Single().ToStackItem().GetBigInteger();
+            return balance;
+        }
+
+        public string GetName(UInt160 scriptHash)
+        {
+            return TestInvoke(scriptHash, "name").Stack.Single().ToStackItem().GetString();
+        }
+
+        public string GetSymbol(UInt160 scriptHash)
+        {
+            return TestInvoke(scriptHash, "symbol").Stack.Single().ToStackItem().GetString();
+        }
+
+        public uint GetDecimals(UInt160 scriptHash)
+        {
+            return (uint)TestInvoke(scriptHash, "decimals").Stack.Single().ToStackItem().GetBigInteger();
+        }
+
+        public BigInteger GetTotalSupply(UInt160 scriptHash)
+        {
+            return TestInvoke(scriptHash, "totalSupply").Stack.Single().ToStackItem().GetBigInteger();
+        }
+
+        public Transaction Transfer(UInt160 scriptHash, KeyPair fromKey, UInt160 to, BigInteger amount, long networkFee = 0)
+        {
+            var sender = fromKey.ScriptHash;
+
+            byte[] script = MakeScript(scriptHash, "transfer", sender, to, amount);
+            Transaction tx = new TxManager(rpcClient, sender)
```

Comment:
I suspect I'm missing something because I'm reading the code not watching it execute in the debugger. How does the signed transaction get submitted back to the blockchain. Have I missed a call to SendRawTransaction?

Reply:
![image](https://user-images.githubusercontent.com/4393234/63075531-a48f4b80-bf64-11e9-9799-fb630ef3395f.png)
>It would be great if ContractClient and Nep5API could just return the transaction to be signed off-chain. That would be a nice bridge for ledgers and light-wallets.

Reply:
So it's the caller's responsibility to call SendRawTransaction. That makes sense - I figured I was missing something. Can we add doc comments to demonstrate the expected calling pattern?

Reply:
Yes, we will have doc and demos for using RPC SDK